extensions/editor.js

```diff
@@ -3,3 +3,14 @@
  */
 import './shared/public-path';
 import './shared/block-category';
+import analytics from '../_inc/client/lib/analytics';
+
+// @TODO Please make a shared analytics solution and remove this!
+if (
+	typeof window === 'object' &&
+	typeof window.Jetpack_Editor_Initial_State === 'object' &&
+	typeof window.Jetpack_Editor_Initial_State.tracksUserData === 'object'
+) {
+	const { userid, username, blogid: blog_id } = window.Jetpack_Editor_Initial_State.tracksUserData;
```

Comment:
Looks great- just as FYI that we need the prop to be named `blog_id`  to be saved in tracks correctly.